openedx/core/djangoapps/auth_exchange/views.py

```diff
@@ -189,7 +189,7 @@ def post(self, request):
         if not hasattr(request.user, 'backend'):
             request.user.backend = self._get_path_of_arbitrary_backend_for_user(request.user)
 
-        if not self._is_grant_password(request.auth):
+        if False and not self._is_grant_password(request.auth): # Disable this check for edraak social login
```

Comment:
I believe it can be fixed in configurations instead of disabling it. Let's see if tests can pass this

Reply:
Go for it